### PR TITLE
[WIP] Feature/make use of showdown

### PR DESCRIPTION
**Pull request by hiroppy**
We can use GitHub's markdown syntax now!!!

see: https://github.com/showdownjs/showdown

### CODE REVIEW

**Comment by abouthiroppy** [comment]
new options: https://github.com/showdownjs/showdown#valid-options

File: resource/js/util/CrowiRenderer.js [hunk 1]
```diff
@@ -96,35 +96,12 @@ export default class CrowiRenderer {
   parseMarkdown(markdown, dom) {
     let parsed = '';
 
-    const markedRenderer = new marked.Renderer();
-    markedRenderer.code = this.codeRenderer;
```
Comment (sotarok) [inline resource/js/util/CrowiRenderer.js]: We have to think about syntax highlight.

Comment (abouthiroppy) [inline resource/js/util/CrowiRenderer.js]: Good catch.

Comment (abouthiroppy) [inline resource/js/util/CrowiRenderer.js]: I'll add highlight.js as an extension;)

**Comment by sotarok** [comment]
We're thinking how about move forward to remark https://github.com/wooorm/remark

**Comment by abouthiroppy** [comment]
OK, I'll try it.

**Comment by sotarok** [comment]
We'll plan to use remark.js as next markdown parser so close it.
Thank you!

**Pull request closed without merging**